src/components/PasswordInput/PasswordInput.tsx

```diff
@@ -86,7 +102,8 @@ export const PasswordInput: React.FC<PasswordInputProps> = (props) => {
             rightContent={additionalRightContent}
             autoComplete={autoComplete ? autoComplete : 'new-password'}
             controlProps={{
-                className: b(),
+                className: b('input-control', controlProps?.className),
+                ...controlProps,
```

Comment:
Spread should be before the className, otherwise it would be overwritten by the controlProps